tests/c21client/test_client.py

```diff
@@ -84,7 +86,7 @@ def test_client_sends_job_results(mock_requests, mocker):
         )
         mock_requests.put(
             f'{BASE_URL}/put_results',
-            json={'success': 'The job was successfully completed'},
```

Comment:
This change doesn't make sense. This line is mocking out what the put_results endpoint returns which was correct before. The change makes the endpoint return the data that is supposed to be sent to it.

Reply:
He is right... @MrTsfree just replace `json=mock_job_result` with `json={'success': 'The job was successfully completed'},` like it was before.

Reply:
I went ahead and rolled this back to `json={'success': 'The job was successfully completed'}`.  Sorry about that!  Check my latest commit for the change.